iroh-rpc-client/src/lib.rs

```diff
@@ -1,20 +1,24 @@
-pub mod client;
-pub mod config;
-pub mod gateway;
-pub mod network;
-pub mod status;
-pub mod store;
-pub use self::config::Config;
-pub use client::Client;
 use iroh_rpc_types::{gateway::GatewayService, p2p::P2pService, store::StoreService, Addr};
-pub use network::{Lookup, P2pClient};
 use quic_rpc::{
     transport::{combined, http2, CombinedChannelTypes, Http2ChannelTypes, MemChannelTypes},
     RpcClient, RpcServer, Service,
 };
+use tracing::info;
+
+pub use crate::config::Config;
+pub use client::Client;
+pub use network::{Lookup, P2pClient};
+pub use quic_rpc::LocalAddr;
 pub use status::{ClientStatus, ServiceStatus, ServiceType, StatusType, HEALTH_POLL_WAIT};
 pub use store::StoreClient;
 
+pub mod client;
+pub mod config;
+pub mod gateway;
+pub mod network;
+pub mod status;
+pub mod store;
+
```

Comment:
Was this moved for a style-guide reason?

Reply:
Ugh, I kind of did that without thinking out of habit.  Seems I'm backed up by the style guide though: https://doc.rust-lang.org/stable/style-guide/items.html#items

> We recommend that imports come before module declarations;

Happy to revert though - IMHO this kind of thing can do whatever unless we configure rustfmt to be strict (it doesn't do this - yet)

Reply:
Nah, no need to change, I was genuinely just curious. Thanks!